lib/WUI/netdev.c

```diff
@@ -220,14 +129,14 @@ uint32_t netdev_init() {
     active_netdev_id = variant8_get_ui8(eeprom_get_var(EEVAR_ACTIVE_NETDEV));
 
     tcpip_init(tcpip_init_done_callback, NULL);
-    netdev_init_esp();
 
     alsockets_adjust();
     return 0;
 }
 
 uint32_t netdev_init_esp() {
-    esp_init(esp_callback_func, 0);
+    // TODO: Is this so simple ???
+    espif_init(&wlan0);
```

Comment:
Well, I'd suppose the initialization can fail and therefore we should propagate the error further.